packages/format-currency/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.0.0 - 2022-04-25
```

Comment:
Did you mean `1.0.1`?

Reply:
Oh yes 🤦

Reply:
Should I create a PR for this or it can be fixed directly in `trunk`?

Reply:
A PR sounds good 👍 We generally never touch `trunk` directly here. 

Happy to approve it quickly for you.

Reply:
Here you go
https://github.com/Automattic/wp-calypso/pull/63044

Reply:
Thanks @manzoorwanijk 🙌